scripts/download_osm_data.py

```diff
@@ -2,399 +2,43 @@
 # SPDX-FileCopyrightText: : 2021 PyPSA-Africa Authors
```

Comment:
I like the PR!

I think that now that it's getting stable, it could be nice to add a docstring at the beginning to state the link to earth-osm and give a description of this rule that is missing.
Not necessarily something long

Reply:
Done